quinn-proto/src/connection/streams/state.rs

```diff
@@ -1553,4 +1581,77 @@ mod tests {
             assert_eq!(client.max_remote[Dir::Bi as usize], 201);
         }
     }
+
+    #[test]
+    fn expand_receive_window() {
+        let mut server = make(Side::Server);
+        let new_receive_window = 2 * 1024 * 1024u32;
```

Comment:
For robustness, let's make this `2 * server.receive_window` or similar, rather than relying on one hardcoded value having a particular relationship to another hardcoded value elsewhere.

Reply:
Done